tox.ini

```diff
@@ -39,6 +33,12 @@ basepython =
 setenv =
     PURE_PYTHON = 1
 
+[testenv:py38-pure]
```

Comment:
@mgedmin pointed me towards [tox's factors](https://tox.readthedocs.io/en/latest/config.html#factors-and-factor-conditional-settings), which let you condense a lot of configuration. Instead of separate sections for `py27-pure`, `py38-pure`, etc, one can simply write:
```ini
[testenv]
setenv =
    THIS_IS_ALWAYS_SET = 1
    pure: ONLY_WHEN_PURE_IS_IN_THE_NAME = 1
```

Reply:
Nice improvement idea, I'll try it.